packages/application-components/src/components/custom-views/custom-view-loader/custom-view-loader.tsx

```diff
@@ -49,6 +49,87 @@ const CustomPanelIframe = styled.iframe`
   border: none;
 `;
 
+function useIFrameInitializationEffect({
```

Comment:
The idea was to use a polling mechanism to check for an element - in this case loading spinner - as a ready indicator for receiving messages inside the iframe. 

The polling would start upon onLoad and finish once the message was sent.